experimental/uefi/app/src/remote_attestation.rs

```diff
@@ -0,0 +1,159 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Server-side implementation of the bidirectional gRPC remote attestation handshake
+//! protocol.
+//!
+//! A simplified version of the implementation from the `grpc_unary_attestation`
+//! crate. TODO(#2741): Refactor this to share more code between the two runtimes.
```

Comment:
Will do some of that before opening this PR, to simplify review

Reply:
done in https://github.com/project-oak/oak/pull/2743

Reply:
Leaving this TODO in as some additional refactoring can probably be done with the AttestationHandler here, which mirrors the `AttestationServer` in the grpc attestation crate. — But redundancy is fairly small, so we can move ahead with this PR for now.